### PR TITLE
Add tests for model loading.

### PR DESCRIPTION
**Pull request by jf514**
1. Created and exposed a utility function for loading models into mjx.
2. Created integration tests for rodent and mouse models.
3. Moved unit tests into `unit` sub-directory in the `tests` directory.

### CODE REVIEW

File: tests/integration/test_model.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    return "models/mouse_with_meshes.xml"
+
+
+# Tests
+def test_rodent_load(rodent_model):
+    load_model(rodent_model)
+
+
+def test_mouse_load(mouse_model):
+    load_model(mouse_model)
```
[the remark this answers is not in the document]
Comment (jf514) [reply tests/integration/test_model.py:38]: This has too much code duplication.

**Review by talmo — APPROVED** [review]
LGTM! Thanks for adding the tests :)

File: tests/integration/test_model.py
Comment (charles-zhng) [inline tests/integration/test_model.py:1]: Just curious--what makes this an integration test?

**Review by charles-zhng — COMMENTED** [review]
LGTM, just had the one question!

**Pull request merged**